src/transformers/models/bridgetower/modeling_bridgetower.py

```diff
@@ -0,0 +1,1006 @@
+# coding=utf-8
+# Copyright 2022 The Intel Labs Team Authors, The Microsoft Research Team Authors and HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PyTorch BridgeTower Model"""
+
+from collections import OrderedDict
+from dataclasses import dataclass
+from typing import Optional, Tuple, Union
+
+import torch
+import torch.utils.checkpoint
+from torch import nn
+
+from transformers import RobertaConfig, RobertaModel
+from transformers.modeling_utils import PreTrainedModel, apply_chunking_to_forward
+from transformers.models.bert.modeling_bert import BertAttention, BertIntermediate, BertOutput, BertPooler
```

Comment:
Maybe it's better to copy the classes themselves, and add `# Copied from ...` but I am not sure here, so let's wait for @ArthurZucker and/or @sgugger to chime in to give their opinion

Reply:
This is the core philosophy of Transformers. All models should be fully independent of each other, so those imports from Roberta/BERT should not be there.

Reply:
Thanks @younesbelkada  and @sgugger - I have handled this and removed all Bert/Roberta imports